files/zh-cn/web/css/@supports/index.md

```diff
@@ -132,7 +149,7 @@ slug: Web/CSS/@supports
 ```css
 @supports ( not ((text-align-last:justify) or (-moz-text-align-last:justify) ){
     … /* 这里的 CSS 代码用来模拟 text-align-last:justify */
-}
+};;
```

Comment:
Seems to be wrong here. This is a CSS rule🤔

Reply:
Ah, I see what's going on -- there's some bad syntax here!